tests/test_ui.py

```diff
@@ -103,8 +103,7 @@ def test_appending_of_initial_html(selenium, app, status, warning):
                     <span class="bar"></span>
                 </div>
                 <div class="rtd__search__credits">
-                    Search By <a href="https://readthedocs.org/">Read the Docs</a> |
-                    In-Doc Search By <a href="https://readthedocs-sphinx-search.readthedocs.io/en/latest/">readthedocs-sphinx-search</a>
+                    Search by <a href="https://readthedocs.org/">Read the Docs</a> &amp; <a href="https://readthedocs-sphinx-search.readthedocs.io/en/latest/">readthedocs-sphinx-search</a>
```

Comment:
Why didn't this fail on the PR when we added it?

Reply:
We forgot to update the minified files there.